airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,361 @@
+| slug | varchar(255) | yes | | |
+| email | varchar(255) | | | index |
+| initial_setup_complete | boolean | yes | false | |
+| anonymous_data_collection | boolean | | true | |
+| news | boolean | | false | | |
+| security_updates | boolean | | false | |
+| display_setup_wizard | boolean | | true | Artem suggested to default this field to `true`. |
+| tombstone | boolean | | false | index |
+
+### `WORKSPACE_NOTIFICATION`
+
+YAML reference:
+- [Notification](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/Notification.yaml)
+
+This table has many-to-one relationship with the `WORKSPACE` table. One workspace can have multiple notifications.
+
+This is the base table for notifications. Each notification type will have its own child table (e.g. `SLACK_NOTIFICATION`).
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| notification_type | varchar(50) | yes | | Index. String enum: "slack". |
+| send_on_success | boolean | | false | index |
+| send_on_failure | boolean | | false | index |
+
+### `SLACK_WORKSPACE_NOTIFICATION`
+
+YAML reference:
+- [SlackNotificationConfiguration](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SlackNotificationConfiguration.yaml)
+
+This table has one-to-one relationship with the `WORKSPACE_NOTIFICATION` table. Each `SLACK_WORKSPACE_NOTIFICATION` belongs to one `WORKSPACE_NOTIFICATION`.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| notification_id | char(36) | yes | | Foreign key. Unique index to enforce 1:1 mapping. |
+| webhook | varchar(2048) | yes | | |
+
+### `SOURCE_DEFINITION`
+
+YAML reference:
+- [StandardSourceDefinition](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardSourceDefinition.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | | |
+| docker_repository | varchar(255) | yes | | Index (ideally this should be unique, but we have duplicates for marketing purpose). |
+| docker_image_tag | varchar(50) | yes | | |
+| documentation_url | varchar(255) | yes | | |
+| icon | varchar(63) | | | |
+
+### `SOURCE_CONNECTION`
+
+YAML reference:
+- [SourceConnection](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SourceConnection.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | |
+| source_definition_id | char(36) | yes | | Foreign key. Index. |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| configuration | jsonb | yes | | Must be a valid json string. |
+| tombstone | boolean | | false | index | |
+
+### `DESTINATION_DEFINITON`
+
+YAML reference:
```

Comment:
`id`?

Reply:
Same. The `destinationDefinitionId` is replaced with `id`.